salttesting/case.py

```diff
@@ -321,7 +321,7 @@ def detach_from_parent_group():
                     break
         tmp_file.seek(0)
 
-        if sys.version_info.major > 2:
+        if sys.version_info > (2,):
```

Comment:
You will need to make this `>= (3,)` to maintain the previous logic. `(2, 4) > (2,)`...